KenticoCloud.Delivery/InlineContentItems/InlineContentItemsProcessor.cs

```diff
@@ -102,19 +102,16 @@ private string GetFragmentText(object inlineContentItem)
                 return inlineContentItemResolver(inlineContentItem);
             }
 
-            var data = new ResolvedContentItemData<object> { Item = inlineContentItem };
             if (_inlineContentItemsResolvers.TryGetValue(typeof(object), out var defaultContentItemResolver))
             {
-                return defaultContentItemResolver(data);
```

Comment:
I consider this a breaking change even though we're not changing the contract...